deps/libsodium.cmake

```diff
@@ -23,9 +24,10 @@ if (WIN32)
            message("libsodium = ${libsodium_lib_path}")
     else()
            FetchContent_Declare (
-		   libsodium
-		   URL	https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable-mingw.tar.gz
-		   )
+                   libsodium
+                   URL	https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable-mingw.tar.gz
+                   URL_HASH MD5=110bbb6ca2322a0c2b3f639432e15cea
```

Comment:
Was this added because of a tool upgrade or because we specifically added it?

Reply:
I just added because it seemed like a good idea